packages/lesswrong/lib/collections/comments/views.js

```diff
@@ -337,8 +337,9 @@ Comments.addView('nominations2018', function ({userId, postId, sortBy="top"}) {
     }
   };
 });
+// Filtering comments down to ones that include "nominated for Review" so further sort indexes not necessary
 ensureIndex(Comments,
-  augmentForDefaultView({ userId:1, postId: 1, nominatedForReview: 1}),
+  augmentForDefaultView({ nominatedForReview: 1}),
```

Comment:
I think this does actually need to have `userId` and `postId` (because mongodb is terrible and can't do index joins), it just can't have them precede `nominatedForReview`.
```suggestion
  augmentForDefaultView({ nominatedForReview: 1, userId: 1, postId: 1 }),
```